### ISSUE TITLE
Turbine unable to track dll and opt files 

### ISSUE THREAD

**Issue by anujad95** [CCSI-Toolset/FOQUS#635]
Recently there has been an issue with running Aspen models containing dll and opt files, in FOQUS. The node simulation fails in the first run with a "Turbine simulation error". When the 'test' sub directory of FOQUS work directory was opened to check the set of files that turbine accesses to run the simulation, only the Aspen file was found, not the dll and opt files. That's the reason it gave the simulation error. 
In order to work around this issue for the time being, turbine had to be restarted, and the dll, opt files had to be re added to json, which was  re uploaded to turbine. Then too, the node had to be run 2-3 times to make the simulation work.

**Comment by eslickj** [comment]
I'm not sure about the dll file part but I may have this issue.  With ACM I ran a simulation and the first time it failed.  Here are the details.

```
"[\"event=setup,consumer=f22e67bd-47d5-4345-82d5-cbb7639e2a34\", 
\"event=error,consumer=f22e67bd-47d5-4345-82d5-cbb7639e2a34,
msg=\\\"copyFilesToDisk Failed to copy files to working directory: The file \"test\\\\260\\\\BFB_v11_FBS_01_26_2018.acmf\" already exists. 
StackTrace:    
  at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)\\r\\n   
  at System.IO.File.InternalCopy(String sourceFileName, String destFileName, Boolean overwrite, Boolean checkHost)\\r\\n   
  at System.IO.File.Copy(String sourceFileName, String destFileName)\\r\\n   
  at Turbine.Consumer.AspenTech.AspenSinterConsumer.copyFilesToDisk(IJobConsumerContract job) in c:\\\\Users\\\\boverhof\\\\Documents\\\\Visual Studio 2012\\\\Projects\\\\turb_sci_gate\\\\Master\\\\Turbine.Consumer.AspenTech\\\\AspenSinterConsumer.cs:line 121\\r\\n
  at Turbine.Consumer.SimSinter.SinterConsumerRun.DoSetup(IJobConsumerContract job) in c:\\\\Users\\\\boverhof\\\\Documents\\\\Visual Studio 2012\\\\Projects\\\\turb_sci_gate\\\\Master\\\\Turbine.Consumer.SimSinter\\\\SinterConsumerRun.cs:line 398\\\"\"]"```

**Comment by anujad95** [comment]
Yes, me and Josh Morgan got a similar error while running Aspen Plus simulations. @boverhof this is an issue with turbine itself, or SimSinter too?

**Comment by ksbeattie** [comment]
This might be fixed in the new Dec 2019, 2.2.0 version of turb_sci_gate.  @jmorgan29 & @anujad95 will re-test and see if this still exists.

**Comment by jmorgan29** [comment]
I just installed the new version of Turbine, uploaded a few models to Turbine, and verified that they worked correctly on the first attempt. I would recommend closing the issue after @anujad95 verifies.
